### ISSUE TITLE
Issue with list commands in 2.1

### ISSUE THREAD

**Issue by djarnis73** [tonivade/claudb#158]
Hi

After attempting to bump our claudb dependency from 2.0.1 to 2.1 one of our unit test started failing on what looks like a bug somewhere in `lpush`/`rpush`/`lpop`/`rpop` commands.

Simple reproducer (can be copied directly into [ClauDBServerTest](https://github.com/tonivade/claudb/blob/master/lib/src/test/java/com/github/tonivade/claudb/ClauDBServerTest.java)):

```
@Test
public void testPushPull() {
  execute(jedis -> {
    long push = jedis.rpush("key", "val1", "val2");
    String pop1 = jedis.rpop("key");
    String pop2 = jedis.rpop("key");
    assertThat(push, is(2l));
    assertThat(pop1, equalTo("val2"));
    assertThat(pop2, equalTo("val1"));
  });
}
```

It fails on the last assert where `pop2` is `null` instead of expected val1.

I checked the [changes](https://github.com/tonivade/claudb/compare/2.0.1...2.1) and the `lpush` and `rpush` commands were both changed, so I would expect a bug was introduced here.

**Comment by tonivade** [comment]
hi @djarnis73 I released a snapshot for release 2.1.1, can you take a look if it works for you? 

```
<dependency>
  <groupId>com.github.tonivade</groupId>
  <artifactId>claudb</artifactId>
  <version>2.1.1-SNAPSHOT</version>
</dependency>
```

**Comment by djarnis73** [comment]
Hi @tonivade, I can confirm that our test no longer fails with 2.1.1-SNAPSHOT. Thanks a lot for the fix.

On a side note, I would really love to see a dependency update of the resp-server, to get rid of the netty-all transitive dependency.

**Comment by tonivade** [comment]
Ok, I will release version 2.1.1 ASAP

About resp-server without netty transitive dependencies. My idea is to create another release 2.2 with this change, and include the fix made in version 2.1.1.

**Comment by tonivade** [comment]
the new release 2.1.1 is available https://github.com/tonivade/claudb/releases/tag/2.1.1
